frontend/assets/javascripts/src/main.js

```diff
@@ -20,11 +21,10 @@ require([
     'src/modules/identityPopup',
     'src/modules/identityPopupDetails',
     'src/modules/metrics',
-    'src/modules/patterns',
-    // Add new dependencies ABOVE this
-    'raven'
```

Comment:
This was pointing to Raven which is the sentry client, it's an implied global variable, hence the awkward comment. It's not moved to `frontend/assets/javascripts/src/modules/raven.js` below.